docs/source/references/modules.rst

```diff
@@ -12,6 +12,25 @@ metrics.
 .. autoclass:: torchmetrics.Metric
     :noindex:
 
+.. note::
+
+    Metric states are being synchornized by default in a distributed setting when using :meth:`~torchmetrics.Metric.state_dict`.
```

Comment:
Maybe it is worth mentioning that the state dict by default is empty because `persistent=False`

Reply:
Done !